dataminer-overview/DevOps/DataMiner_Devops_Professionals/Benefits_DevOps_Professional_Program.md

```diff
@@ -11,23 +11,120 @@ By creating value for the DataMiner community, you earn [**DevOps Points**](xref
+        <td style="text-align:center;"></td>
+        <td style="text-align:center;"></td>
+      </tr>
+      <tr>
+        <td style="text-align:left;">
+          To further support and accelerate you and to empower you with knowledge, we will provide you with exclusive content. You will be the first to know, the first to weigh in, and the first to benefit from the transformational power of DataMiner.
+        </td>
+        <td style="text-align:center;">-</td>
+        <td style="text-align:center;">&#x2714;</td>
+        <td style="text-align:center;">&#x2714;</td>
+      </tr>
+      <tr>
+        <td style="text-align:left;"><strong>Join developer reviews</strong></td>
+        <td style="text-align:center;"></td>
+        <td style="text-align:center;"></td>
+        <td style="text-align:center;"></td>
+      </tr>
+      <tr>
+        <td style="text-align:left;">
+          Whenever we work on developing evolutions of DataMiner that are important for you, our development squads will include you in regular review meetings, so that you can provide feedback in real time and achieve your goals more quickly.
+        </td>
+        <td style="text-align:center;">-</td>
+        <td style="text-align:center;">-</td>
+        <td style="text-align:center;">&#x2714;</td>
+      </tr>
+      <tr>
+        <td style="text-align:left;"><strong>Exclusive option to convert your Dojo credits</strong></td>
+        <td style="text-align:center;"></td>
+        <td style="text-align:center;"></td>
+        <td style="text-align:center;"></td>
+      </tr>
+      <tr>
+        <td style="text-align:left;">
+          Dojo activity typically results in credits that you can spend in the <a href="https://community.dataminer.services/swag-shop/">Swag Shop</a> on DataMiner Dojo. However, as an accredited DataMiner DevOps Professional, you can also convert these into free-of-charge <a href="https://community.dataminer.services/product/dataminer-credit/">DataMiner credits</a> to further accelerate your ambitions.
+        </td>
+        <td style="text-align:center;">&#x2714;</td>
+        <td style="text-align:center;">&#x2714;</td>
+        <td style="text-align:center;">&#x2714;</td>
+      </tr>
+    </tbody>
+  </table>
+</figure>
 
-- Free personal DataMiner licenses
-
-- Access to DIS source code
-
-- Priority community support
-
-- Access to exclusive content
-
-- Option to join developer reviews
-
-- Exclusive option to convert your DevOps Points to [DataMiner credits](xref:Pricing_Usage_based_service#usage-terms) or [Professional Service credits](xref:Professional_service_credits)
-
-Learn how to earn DevOps Points: [Accumulating DevOps Points](xref:DevOps_Points)
+Curious how you can earn DevOps Points? Refer to [Accumulating DevOps Points](xref:DevOps_Points).
 
 > [!TIP]
 > You can find a more extensive overview of all benefits on [DataMiner Dojo](https://community.dataminer.services/dataminer-devops-professional-program/#benefits).
```

Comment:
Is this tip still needed, given that there is now a more extensive overview here? If not, we'd better remove it.